src/components/Codeblock.tsx

```diff
@@ -1,10 +1,11 @@
-import React, { useState } from "react"
+import React, { useEffect, useState } from "react"
```

Comment:
We haven't really discussed formally yet, but I've been removing the `React` imports since they're no longer needed with our new ts configuration.

ie just
```suggestion
import { useEffect, useState } from "react"
```